developers/smart-contracts/lock-api/README.md

```diff
@@ -36,6 +38,7 @@ This function can be invoked to purchase a key (NFT membership).
 To purchase a membership, the caller should pass the number of tokens to be transfered (an ERC20 approval should have been initiated earlier), and this amount can be larger than the actual `keyPrice`.
 It is possible to purchase a membership on behalf of another user by setting a different recipient address.
 The referrer address is the address that will receive some UDT governance tokens as part of this transaction (if applicable).
+A key manager can also be set directly during the purchase, allowing a different account to manage the key while the recipient still benefit from a full membership.
```

Comment:
Maybe a link to the "Access Control" page could help here? (this is where we explain the roles)